### PR TITLE
Recipe group project

### PR DESCRIPTION
**Pull request by dfsalam**
### Inventory shopping list, a shopping list, but only taking into consideration a chosen recipe and inventory.

### CODE REVIEW

File: spec/requests/inventories_spec.rb [hunk 1]
```diff
@@ -0,0 +1,93 @@
+require 'rails_helper'
```
Comment (BenMukebo) [inline spec/requests/inventories_spec.rb:1]: - [ ] Greate job using Capybara testing libraries for writing integration test now let's add unit test as well using RSpec 👍🏻

File: config/routes.rb [hunk 2]
```diff
@@ -0,0 +1,30 @@
+Rails.application.routes.draw do
+  devise_for :users
+  resources :inventories, only: [:index, :show, :new, :create, :destroy]
+  resources :inventory_foods, only: [:create,:destroy]
+  resources :foods, only: [:new, :create, :destroy]
```
Comment (BenMukebo) [inline config/routes.rb:5]: image

- [ ] Kindly let's address this by adding foods index route. 👍🏻

**Review by BenMukebo — CHANGES_REQUESTED** [review]
## Hi @dfsalam, @kensteph, @V0pe,

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


### To Highlight 🎉

- Git flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️
- The application has a login and Registration page, built with the Devise gem.
- Fix N+1 queries problem from happening. ✔️ 
- Use Capybara testing libraries. ✔️ 

## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers :clinking_glasses:  and Happy coding!!! :dancers:💻  

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @BenMukebo when you comment so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by kensteph** [comment]
Hi reviewer 👋!
We want to draw your attention to this requirement : 

image

This requirement is only for groups with less than 3 members.

Thank you for your time.

**Review by brytebee — APPROVED** [review]
Hello Team 👋 ,

------

### Highlights💡
- Good commit message.✔️ 
- Good PR Summary.✔️
- Great UI design.✔️
- App functions as it should.✔️

------

In my opinion, your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 



Good luck! 🎉 

------

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**